.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+.env
+.env.test
```

Comment:
As we do not keep any sensitive data here for now let's add this to our repo for simplicity

Reply:
fixed in 78cfdcd